app/src/main/java/com/example/moviedetailsapplication/network/MoviesInterface.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.moviedetailsapplication.network
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface MoviesInterface {
+
+    @GET("discover/movie?sort_by=popularity.desc&api_key=e7d826d84097c8429e815cd8ab31c7a0")
```

Comment:
Please check `interceptors` for OkHTTP. we can dynamically add API key to request. we should not hard code like this